Dockerfile.ansible

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:14.04
```

Comment:
adds a dockerfile using ansible, to test the ansible role that checkbot will eventually pull in